LiteCore/Database/SequenceTracker.cc

```diff
@@ -183,7 +198,7 @@ namespace litecore {
                 nextEntry = next(entry);
                 if (!entry->isPlaceholder()) {
                     // moves entry!
-                    _documentChanged(entry->docID, entry->revID,
+                    _documentChanged(entry->collection, entry->docID, entry->revID,
```

Comment:
Here, for example, if we store the collection in SequenceTracker itself, "entry->collection" can be replaced by this->collection. Logically, collection is welded into the Tracker, const/immutable.